frontend/src/pages/AccountSend.vue

```diff
@@ -292,21 +293,60 @@ function useSendForm() {
   }
 
   const isNativeToken = (address: string) => getAddress(address) === NATIVE_TOKEN.value.address;
-  const getMinSendAmount = (tokenAddress: string) => {
+
+  const getNativeTokenMinSendAmountFallback = (chainId: number): number => {
+    switch (chainId) {
+      case 137: return 1.0; // Polygon
+      default: return 0.01; // everything else
+    }
+  }
+
+  const getTokenMinSendAmountFallback = (
+    tokenAddress: string,
+    chainId: number
+  ): number => {
+    switch (chainId) {
+      case 137: // Polygon
+        if (tokenAddress === '0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619') {
+          return 0.001; // weth token minimum
+        } else {
+          return 3; // stablecoin token minimum
+        }
+      default:
+        // Mainnet, Rinkeby, and other networks have higher ETH and stablecoin minimums
+        // due to higher fees
+        return 100
```

Comment:
This number should probably be revised. The live values are at least 5x as high. In the event that the backend went down and we didn't have live minSends beind sent, we'd probably still want to enforce the high estimates

Reply:
I'll defer to you and @apbendi on good fallback values here

Reply:
Sorry this diff is outdated now and I'm not clear what was being asked. Is my input still needed on something here?

Reply:
Umm I'm not certain either haha, deferring to @davidlaprade 😅

Reply:
I revised the numbers so they were close to recent minSend values from the backend. No further action needed. Thanks, guys!